app/components/avo/panel_component.rb

```diff
@@ -7,14 +7,16 @@ class Avo::PanelComponent < ViewComponent::Base
 
   renders_one :tools
   renders_one :body
+  renders_one :sidebar
```

Comment:
Not sure if we should have a slot for sidebar here on the panel component or render it some other way